src/sidebar/reducers/annotations.js

```diff
@@ -173,17 +173,16 @@ var update = {
 
   UPDATE_ANCHOR_STATUS: function (state, action) {
     var annotations = state.annotations.map(function (annot) {
-      var match = (annot.id && annot.id === action.id) ||
```

Comment:
It used to be the case that some annotations (existing ones that had not yet been anchored) had only IDs, some had only tags (newly created but not yet saved ones) and some had both (saved annotations that had been anchored).

Every annotation now has a `$tag` as soon as it is added to the store, so we no longer need to check for either an ID or a tag.

Reply:
Ah, I see: "This commit also takes the opportunity to remove the logic to update annotation tags when handling this action since it is no longer required." This caused me a fair bit of confusion (since the change seemed to have nothing to do with coalescing updates), if this could have been done as a separate PR or at least a separate commit it might have been easier to follow.